apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -17,7 +17,6 @@ const USER_EDITABLE_SECTION_PROPS = [
   'ttsAutoplayEnabled',
   'participantType',
   'courseId',
-  'scriptId',
```

Comment:
Just checking: this is redundant with `unitId` right?

Reply:
Yup - Just moving over to unitId